### PR TITLE
Update workflows v1

### PR DESCRIPTION
**Pull request by awharn**
Updates zowe-v1-lts workflows to supported versions and remove Node 14 support

### CODE REVIEW

File: .github/workflows/audit.yml
Comment (zFernand0) [inline .github/workflows/audit.yml:1]: |If it's ok, can we add the `zowe-v?-lts` branch to the following workflows:
- audit
- changelog
- linter (just needs to change the `v1` with `v?` 😋 

Here is an example
```yaml
on:
  pull_request:
    branches:
      - master
      - zowe-v?-lts
```

**Review by zFernand0 — CHANGES_REQUESTED** [review]
Changes LGTM!
Just left one small comment 😋

**Review by t1m0thyj — APPROVED** [review]

File: .github/workflows/audit.yml [hunk 1]
```diff
@@ -14,12 +14,12 @@ jobs:
...
     steps:
     - name: Checkout
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
-    - name: Use Node.js 14.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v3
       with:
-        node-version: 14.x
```
Comment (zFernand0) [inline .github/workflows/audit.yml:22]: should we use `lts/*`?

File: .github/workflows/audit.yml
Comment (zFernand0) [reply .github/workflows/audit.yml:1]: this same comment may apply to the master workflows 😋

**Review by zFernand0 — CHANGES_REQUESTED** [review]

**Review by zFernand0 — CHANGES_REQUESTED** [review]
should we add the `zowe-v?-lts` to the `audit` and `changelog` workflows?

**Comment by awharn** [comment]
> should we add the `zowe-v?-lts` to the `audit` and `changelog` workflows?

Wouldn't we want to make that change on the master branch instead of this one? That is the branch's workflows that would apply to zowe-v2-lts, zowe-v3-lts, etc.

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**